contributors/devel/api_changes.md

```diff
@@ -623,8 +623,11 @@ tips](../api.md#v1-conversion-tips))
 via a flag should not create new bugs in unrelated features; because the feature
 is new, it may have minor bugs
   - Support: the project commits to complete the feature, in some form, in a
-subsequent Stable version; typically this will happen within 3 months, but
-sometimes longer; releases should simultaneously support two consecutive
+subsequent Stable version; beta features are required to remain in beta for
```

Comment:
This is overly precise, I think.  Not every feature needs to follow the same timeline?